Rakefile

```diff
@@ -60,16 +77,21 @@ task :analyze do
   sh "set -o pipefail && ! xcodebuild -workspace #{EXAMPLE_WORKSPACE} -scheme \"#{EXAMPLE_SCHEMA}\" -sdk #{BUILD_SDK} -destination \"platform=iOS Simulator,name=iPhone 8\" ONLY_ACTIVE_ARCH=NO analyze 2>&1 | xcpretty | grep -A 5 \"#{ANALYZE_FAIL_MESSAGE}\""
 end
 
+task :eraseDevices do
```

Comment:
This has been found to help in other cases documented online of XCUI tests being flaky in CI.
I didn't want to make it part of the test command as it might be annoying to have devices wiped locally each time.

Reply:
Can we use a snake case name?

Reply:
> documented online

Do you have a link?